### DISCUSSION TITLE
Prevent liquid mode?

### DISCUSSION THREAD

**Discussion by itscvenk** [mpdf/mpdf#1905] [Q&A]
The so called "AI" enabled liquid mode on Adobe Acrobat Reader is more of a **headache** with some of the PDF I generate. It just looks weird and ugly and doesn't make sense

Is there a tag we could add or some other code when generating PDF files so that the liquid mode is not able to apply its blessed "**intelligence**" on it?

Thanks!

**Comment by finwe** [comment]
You have to disable the liquid mode [in the settings of Reader](https://community.adobe.com/t5/acrobat-reader-mobile-discussions/how-to-turn-off-liquid-feature/td-p/12029563). As far as I have found, there is no option to disable it from the PDF.

**Reply by itscvenk** [reply]
thanks for the answer @finwe 

However, randomly, sometimes when I open a PDF document on the mobile, the liquid mode isn't available. It will be interesting to know how that happens so that, when we create the PDF, we can try to do it accordingly in mpdf

Thanks

**Reply by ssieb** [reply]
Maybe that pdf isn't actually text?  If you come across one of those files and can share it, maybe someone can find the answer.
